driver-core/src/main/java/com/datastax/driver/core/TableOptionsMetadata.java

```diff
@@ -183,8 +184,10 @@
       this.extensions = ImmutableMap.copyOf(row.getMap(EXTENSIONS, String.class, ByteBuffer.class));
     else this.extensions = ImmutableMap.of();
 
-    if (is380OrHigher) this.cdc = isNullOrAbsent(row, CDC) ? DEFAULT_CDC : row.getBool(CDC);
-    else this.cdc = DEFAULT_CDC;
+    boolean hasCDCExtension = this.extensions.containsKey(CDC_EXTENSION);
+    if (is380OrHigher)
```

Comment:
We need separate field. this.cdc is a Cassandra CDC. The driver has to work with both Cassandra and Scylla so we need something like this.scyllaCDC as Cassandra CDC and Scylla CDC are separate concepts.

Reply:
Fixed in https://github.com/scylladb/java-driver/pull/35/commits/16bfd04695707cf42ef3b9be347d1a5010aa23cf